lib/ledger_sync/adaptors/ledger_serializer.rb

```diff
@@ -12,9 +12,8 @@ class LedgerSerializer
 
       attr_reader :resource
 
-      def initialize(ensure_inferred_resource_class: true, resource:)
+      def initialize(resource:)
```

Comment:
We had talked about maybe altering this in such a way that this interface doesn't break going forward. Up to you